devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -101,5 +142,172 @@ protected SpanByte SliceGenericBuffer(int start, int length)
 
             return _genericBuffer;
         }
+
+
+        /// <summary>
+        /// Draws a pixel on the screen.
+        /// </summary>
+        /// <param name="x">The x coordinate on the screen.</param>
+        /// <param name="y">The y coordinate on the screen.</param>
+        /// <param name="useWhite">Indicates if color to be used, default yes unless is inverted screen.</param>
+        public void DrawPixel(int x, int y, bool useWhite = true)
+        {
+            if ((x >= Width) || (y >= Height))
+                return;
+
+            // x specifies the column
+            int idx = x + (y / 8) * Width;
+
+            if (useWhite)
+                _genericBuffer[idx] |= (byte)(1 << (y & 7));
+            else
+                _genericBuffer[idx] &= (byte)~(1 << (y & 7));
+
+            //INVERSE COLOR: _genericBuffer[idx] ^= (byte)(1 << (y & 7));
+        }
+
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
+        /// <param name="width"></param>
+        /// <param name="height"></param>
+        /// <param name="bitmap"></param>
+        public void DrawBitmap(int x, int y, int width, int height, byte[] bitmap)
+        {
+            if ((width * height) != bitmap.Length)
+                throw new ArgumentException("Width and height do not match the bitmap size.");
+
+            for (var yO = 0; yO < height; yO++)
+            {
+                for (var xA = 0; xA < width; xA++)
+                {
+                    var b = bitmap[(yO * width) + xA];
+                    byte mask = 0x01;
+                    for (var pixel = 0; pixel < 8; pixel++)
+                    {
+                        DrawPixel(x + (8 * xA) + pixel, y + yO, (b & mask) > 0);
+                        mask <<= 1;
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
+        /// <param name="str"></param>
+        public void DrawString(int x, int y, string str)
```

Comment:
There is a fundamental difference between DrawString used here and Write seen in other similar drivers, the x and y coordinates are scaled in pixels. The difficulty with this approach is that the user must know the font size and calculate the position.
Might I propose that you keep DrawString exactly as-is by correct the comments to say x,y are pixel coordinates,
and create an additional function Write with the same arguments but which scales the x,y into text position coordinates ?

Reply:
something like this,
        /// <summary>
        /// Writes a text message on the screen with font in use.
        /// </summary>
        /// <param name="x">The x pixel-coordinate on the screen.</param>
        /// <param name="y">The y pixel-coordinate on the screen.</param>
        /// <param name="str">Text string to display.</param>
        /// <seealso cref="Write"/>
        public void DrawString(int x, int y, string str)
        {
            byte[] bitMap = GetTextBytes(str);
            DrawBitmap(x, y, bitMap.Length / Font.Height, Font.Height, bitMap);
        }

        /// <summary>
        /// Writes a text message on the screen with font in use.
        /// </summary>
        /// <param name="x">The x text-coordinate on the screen.</param>
        /// <param name="y">The y text-coordinate on the screen.</param>
        /// <param name="str">Text string to display.</param>
        /// <seealso cref="DrawString"/>
        public void Write(int x, int y, string str)
        {
            byte[] bitMap = GetTextBytes(str);
            DrawBitmap((x * this.Font.Width), (y * this.Font.Height), (bitMap.Length / Font.Height), Font.Height, bitMap);
        }

Reply:
Added the updates, will see how the PR is merged. At this time I do not have a good method for large text display.  I have some working code that uses several methods and uses DrawRectangle vs DrawPixel to make larger size fonts.  It may be too slow for some boards.  So any ideas suggestions are welcome in that area.